.travis.yml

```diff
@@ -22,3 +23,10 @@ addons:
 script:
   - bazel clean && bazel build //...
   - bazel clean && bazel test //...
+
+  # Verify that all source files are correctly formatted.
+  - gofmt -d -e -l .
+
+  - go clean -i
+  - go get -u -t ./...
+  - go build -a -v -race ./... 
```

Comment:
Given that we're using bazel for stuff, should this be a bazel build instead of a go build? I guess both should work for build, but it's unlikely "go test" is going to keep working by itself for long.

Reply:
`go test` doesn't work right now. We want to ensure that `go build` works on its own so that this lib can be vendored into pure go projects

Reply:
The point of `go build` working is so that projects can vendor this using traditional Go tooling.  Those also typically prune tests, so it's less critical that `go test` work, but that's the easiest way to get Go coverage, so it might be worth a little effort.

Reply:
Unit tests can achieve compatibility pretty easily, and tests which exercise the file system can do so with a little effort (e.g. by using `testdata` and by taking arguments in the test which take the result of a `gendep` to receive references to files when run under bazel).